src/sage_setup/autogen/interpreters/__init__.py

```diff
@@ -174,7 +159,7 @@ def build_interp(interp_spec, dir):
         write_if_changed(path, method())
 
 
-def rebuild(dirname, force=False):
+def rebuild(dirname, force=False, interpreters=None, distribution=None):
```

Comment:
If you're adding arguments, can you add the missing `INPUT` block that explains what they do?

Reply:
done in 12c04b8